src/generators/operationGenerator.ts

```diff
@@ -482,11 +492,42 @@ function getOperationParameterSignatures(
       return acc;
     }, []);
 
+    // sort parameters to match the signature ordering
```

Comment:
Awesome! Thanks for doing this 😄!

Maybe it would help readability if this is extracted to its own `orderParameters` function

Reply:
Good suggestion, I'll update this.

Reply:
Updated! I also moved the sort call to be after the required parameter groups are included. Turns out they do appear in the operation.request[x].parameters list, so the same logic works for the groups as well.